### PR TITLE
Update native-builder.md

### PR DESCRIPTION
**Pull request by aramlawi**

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
@aramlawi please sign the [CLA](https://www.clahub.com/agreements/mendix/docs).

File: content/refguide/native-builder.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ The `Build` command builds the JavaScript bundles and assets, creates a build on
 If you already ran `prepare`, this is an example of a `build` command:
 
 ```bash
-native-builder.exe build --project-name "CoolApp" --app-version "1.0.0" --build-number 1
+native-builder.exe build --runtime-url "https://app.mendixcloud.com" --project-name "CoolApp" --app-version "1.0.0" --build-number 1
```
Comment (jogboms) [inline content/refguide/native-builder.md]: Even though this is a valid example, it not entirely necessary as the `runtime-url` is not considered mandatory for this command.

File: content/refguide/native-builder.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ native-builder.exe build --project-name "CoolApp" --app-version "1.0.0" --build-
 | `--app-icon-path`           | Absolute path to the app icon.                                            | `C:\MyAppIcon.png`                                  |
 | `--app-round-icon-path`     | Absolute path to the app round icon, specific to Android.                 | `C:\MyAppRoundIcon.png`                             |
 | `--app-splash-screen-path`  | Absolute path to the app splash screen image.                             | `C:\MyAppSplash.png`                                |
-| `--java-home`               | Absolute path to the directory where Java executable is located.          | `C:\Program Files\Java\jdk-10.0.1`                  |
+| `--java-home`               | Absolute path to the directory where Java executable is located.          | `C:\Program Files\AdoptOpenJDK\jdk-11.0.3.7-hotspot`                  |
```
Comment (jogboms) [inline content/refguide/native-builder.md]: This path is user-defined. It could be anything

**Comment by ConnorLand** [comment]
I am closing this PR, according to how our developer has weighed in.

**Pull request closed without merging**